src/__tests__/jest/reducers/articleReducer.test.js

```diff
@@ -0,0 +1,33 @@
+import expect from "expect";
+import articleReducer from "../../../redux/reducers/articleReducer";
+import { VIEW_ARTICLE, SERVER_ERROR } from "../../../redux/action_types";
+
+const initialState = {
+  viewArticle: {
+    article: {},
+    error: {}
+  }
+};
+
+const action = { payload: {} };
+
+describe("Article reducer test", () => {
+  it("should return initial test when no test", () => {
+    expect(articleReducer(initialState, action)).toEqual(initialState);
+  });
+});
+
+describe("Article reducer test", () => {
+  it("should handle article reducer", () => {
```

Comment:
Make this more descriptive. What exactly are we handling?

Reply:
@kimotho-njoki I have made it more descriptive